app/assets/javascripts/mpa/processors/peptide/ECPeptideProcessor.ts

```diff
@@ -0,0 +1,39 @@
+import {ProcessedPeptideContainer} from '../../ProcessedPeptideContainer';
+import {Count} from '../../counts/CountTable';
+import { ECCountTable } from '../../counts/ECCountTable';
+
+export namespace ECPeptideProcessor
+{
+    export function process(processedPeptides: ProcessedPeptideContainer): ECCountTable 
```

Comment:
This function is very similar to the `process` function of the `GOPeptideProcessor`. Maybe we can merge these?

Reply:
Will do.